mysql-test/suite/rpl/t/rpl_killed_ddl.test

```diff
@@ -162,12 +162,12 @@ let $rpl_diff_statement= SELECT event_name, event_body, execute_at
                          ORDER BY event_name;
 
 send CREATE EVENT e2
-  ON SCHEDULE AT CURRENT_TIMESTAMP + INTERVAL 1 DAY
+  ON SCHEDULE AT '2020-01-01 00:00:00' + INTERVAL 1 DAY
```

Comment:
2020 is just 3 years from now - it does not look unrealistic.
Funny thing is that Oracle fixed similar problem in its latest minor releases
https://github.com/percona/percona-server/commit/e7a3ccf6b1b1b41e0fa191d26e9e775517c6deee

May be, we can still use `CURRENT_TIMESTAMP` but eliminate the problem on MTR level. Something like
```sql
--let $timestamp= `SELECT CURRENT_TIMESTAMP()`
send_eval CREATE EVENT e2
  ON SCHEDULE AT '$timestamp' + INTERVAL 1 DAY
```